packages/gravity-ui-nunjucks/src/_patterns/05-pages/05-article.njk

```diff
@@ -0,0 +1,29 @@
+{%- from 'organisms-page-intro' import pageIntroContents -%}
+
+{%- extends 'templates-basic-page' -%}
+
+{%- block pageContent %}
+<main>
+  <article class="grav-o-container grav-c-article">
+    <header class="grav-c-page-intro">
+      {{ pageIntroContents(article.title, article.description, article.dateTime, article.img) | safe }}
+    </header>
+    <div class="grav-c-ostentatious-copy">
+      {{ article.copy | safe }}
+    </div>
+  </article>
+  <hr />
+  <aside class="grav-o-container">
+    <ul class="grav-c-related-items">
```

Comment:
You should replace this with `{% include 'organisms-related-items' %}`. That way, if the related items component ever changes, it will be reflected here. Also, it lets Pattern Lab / Fractal create lineage links between those components in the pattern library.

Reply:
Definitely! That was an oversight.